src/dps-email-poller/src/test/DpsEmailPollerApplicationTests.java

```diff
@@ -0,0 +1,31 @@
+package ca.bc.gov.open.pssg.rsbc.dps.dpsemailpoller.health;
+
+import ca.bc.gov.open.pssg.rsbc.dps.dpsvalidationservice.DpsEmailPollerApplication;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+class DpsEmailPollerApplicationTests {
```

Comment:
Can you remove this one, we are not going to test any http endpoint for this one ;-)